IND/MarketIndices/BasketIndices.rdf

```diff
@@ -161,7 +162,7 @@
 			</owl:Restriction>
 		</owl:equivalentClass>
 		<skos:definition xml:lang="en">constituent of a basket of credit risks, such as an event represented in a credit index</skos:definition>
-		<skos:editorialNote xml:lang="en">OTPP Note:: There will be a specific reference entity and a specific instrument per index - and only one instrument per entity, per index. note that a credi index constituent may be in principal also be loans (as with CDS also), in which case some of the terms here would not apply (specifically, the reference security identity if the Reference Entity has not also issued any such), however it&apos;s not clear how this would be treated.</skos:editorialNote>
+		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">There will be a specific reference entity and a specific instrument per index - and only one instrument per entity, per index.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
not sure about the example "such as an event", since we earlier said that such an index consists of instruments or entities not events. Not sure which is right bu there seems to eb an inconsistency.
Overall, is the index representing the risk of an entity/debt defaulting (regardless of the cause), or of it defaulting through a specific type of event?

Reply:
removed that clause, and removed the note, since it wasn't consistent with what we said above about the index